src/mg/PAMI/Message/Event/Factory/Impl/EventFactoryImpl.php

```diff
@@ -52,18 +52,13 @@ class EventFactoryImpl
     /**
      * This is our factory method.
      *
-     * @param string $message Literall message as received from ami.
+     * @param string $message Literal message as received from ami.
      *
      * @return EventMessage
      */
     public static function createFromRaw($message)
     {
         $eventStart = strpos($message, 'Event: ') + 7;
-/*
```

Comment:
Is there a specific reason that it is commented out? If it is not required any more, why not just delete it?

If you ever need it again, well, that's where git comes in.